eoscostarica/backend-boilerplate
# chore: add wallet service

### What does this PR do?
- Add wallet service
- Updated env configuration
- Update docker-compose.yaml configuration 

### How should this be tested?
- run the below command `make run`
- open `wallet/postman-collection.json` file with postman
- use the `/v1/wallet/create` request
- it should produce a new file `wallet_data/${accountname}.wallet`
